Libraries/LibGUI/TextDocument.h

```diff
@@ -44,7 +44,7 @@ namespace GUI {
 struct TextDocumentSpan {
     TextRange range;
     Color color;
-    Optional<Color> background_color;
+    Color background_color;
```

Comment:
This breaks functionality. If the `background_color` is not assigned, we simply use transparent.
This will make it always use black instead.

Reply:
Upsi, dupsi :-) I guess that's not what I wanted to achieve. Hm, I'm thinking how to avoid this compilation error I got then, but maybe I have a look into that later.